pkg/quota/v1/evaluator/core/resource_claims.go

```diff
@@ -153,7 +153,7 @@ func toExternalResourceClaimOrError(obj runtime.Object) (*resourceapi.ResourceCl
 			return nil, err
 		}
 	default:
-		return nil, fmt.Errorf("expect *resource.ResourceClaim or *v1alpha2.ResourceClaim, got %v", t)
+		return nil, fmt.Errorf("expect *resource.ResourceClaim or *v1beta1.ResourceClaim, got %v", t)
```

Comment:
I wonder whether we should make this future-proof:
```suggestion
		return nil, fmt.Errorf("expect %T or %T, got %v", &resourceapi.ResourceClaim{}, &resourceinternal.ResourceClaim{}, t)
```

Reply:
It is better than I made before.